packages/components/inputs/localized-rich-text-input/src/localized-rich-text-input.tsx

```diff
@@ -162,6 +247,7 @@ const LocalizedRichTextInput = (props) => {
           >
             <LocalizedInputToggle
               isOpen={areLanguagesOpened}
+              // @ts-ignore
```

Comment:
Is this still useful here? If yes, would be nice to have a little comment for having it there.